f2/src/locales/fr.json

```diff
@@ -167,24 +201,14 @@
   "timeFramePage.startDateExample": "Format: mois-jour-année, par exemple: 04-28-2019",
   "timeFramePage.title": "Quand la fraude a-t-elle eu lieu?",
   "timeframePage.nextButton": "Étape suivante : Description de la fraude",
-  "whatHappenedPage.nextButton": "Étape suivante : Renseignements sur le suspect",
-  "whatHappendPage.nextStep": "Ensuite, on vous demandera qui pourrait être impliqué.",
   "whatHappenedPage.backButton": "Retourner aux dates de la fraude",
   "whatHappenedPage.hint": "Racontez-nous l'histoire dans vos propres mots",
   "whatHappenedPage.intro1": "Ce qui s'est passé n'est pas de votre faute. Les fraudeurs utilisent plusieurs techniques pour obtenir ce qu'ils veulent.",
+  "whatHappenedPage.nextButton": "Étape suivante : Renseignements sur le suspect",
   "whatHappenedPage.summary": "Que s'est-il passé?",
   "whatHappenedPage.title": "Décrivez ce qui s'est passé",
-  "evidencePage.title": "Preuve à l'appui",
-  "evidencePage.intro": "Ce n'est pas grave si vous ne conservez aucune documentation de l'événement. Vous pouvez passer à la section suivante.",
-  "evidencePage.details": "Pensez à inclure des détails tels que :",
-  "evidencePage.detail1": "Captures d'écran de pages Web, conversations ou fils de discussion.",
-  "evidencePage.detail2": "Images de reçus, de transactions ou de bons de livraison.",
-  "evidencePage.detail3": "Enregistrement des demandes formulées par le suspect.",
-  "evidencePage.addFileButtom": "Ajouter un fichier",
-  "evidencePage.removeFileButton": "Supprimer le fichier",
-  "evidencePage.nextButton": "continuer",
-  "evidencePage.backButton": "Retour à qui pourrait être impliqué",
-  "evidencePage.nextPage": "Ensuite, il vous sera demandé votre emplacement.",
```

Comment:
Little concerned about this as I have to run the extract command, hoping no translation is lost for other dev :(

Reply:
@sastels  I have synced my local files with master and ran npm extract so this should not cause any conflicts when merged anymore

Reply:
looks like you still have those lines in line 50 - 61 of your file, so they were duplicated.